controlplane/controlplane.go

```diff
@@ -125,6 +125,11 @@ func (lb *LoadBalancer) Start(upstreamCh <-chan []string) error {
 	return nil
 }
 
+// Healthy returns true if at least one upstream is available.
+func (lb *LoadBalancer) Healthy() (bool, error) {
```

Comment:
even though it's trivial, it's nice to add a simple unit-test for it (or extend the existing tests), so that we run `unit-tests-race` and check for data races

Reply:
added `Healthy` calls to the existing test.